### DISCUSSION TITLE
如何使页脚的内容重复加载呢？（例如 在页脚添加计数统计）

### DISCUSSION THREAD

**Discussion by zkeq** [spencerwooo/onedrive-vercel-index#467] [Q&A] [answered]
我使用到的是一个 访问人数的计数统计：  [ 不蒜子 - 极简网页计数器 (busuanzi.ibruce.info)](https://busuanzi.ibruce.info/)

示例站点：https://cdn.dream-plan.cn/zh-CN/

我在 `footer` 添加的代码
```js
<script async src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
本文总阅读量<span id="busuanzi_value_page_pv"></span>次 
本站总访问量<span id="busuanzi_value_site_pv"></span>次 
本站总访客数<span id="busuanzi_value_site_uv"></span>人
```

但是它只在第一次访问的时候有效

![1](https://user-images.githubusercontent.com/62864752/154722939-06d6471c-8b39-43ac-a007-668871b8eb51.png)

当我尝试进入某个文件夹的时候，它会变成这样。

![2](https://user-images.githubusercontent.com/62864752/154723000-90d10384-df3c-4ae5-b030-124596c3d765.png)

我想是可能因为引入的计数 `js` 没有在更换页面的时候重复加载导致的？

![3](https://user-images.githubusercontent.com/62864752/154723236-efca9879-3f93-4657-bfbb-120d965a5421.png)

**Comment by myl7** [comment, marked as answer]
不必局限于配置文件，你可以直接修改 `components/footer.tsx` 实现具体的功能，相信 react 生态下也有相应工具，直接引入即可。

**Reply by zkeq** [reply]
感谢，尽管我对 react 不是很熟悉 

但这启发了我，最终我选择使用一个 `iframe` 解决了此问题。

https://cdn.dream-plan.cn/

**Reply by waleraye** [reply]
> 谢谢，虽然我对反应不是很是
> 
> 今天启发了我，最终我选择使用一个`iframe`解决方案来解决这个问题。
> 
> https://cdn.dream-plan.cn/

你好,现在是怎么修改的呀?

**Reply by zkeq** [reply]
> > 谢谢，虽然我对反应不是很是
> > 今天启发了我，最终我选择使用一个`iframe`解决方案来解决这个问题。
> > https://cdn.dream-plan.cn/
> 
> 你好,现在是怎么修改的呀?

你好，我的页脚代码是这样写的：

```js
<iframe frameborder=0  height=25px marginwidth=0 scrolling=no src="/count.html"></iframe>
```

`/public/count.html` 文件里的内容是这个

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <script async src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    <title>count</title>
</head>
<body>
<style>
    h2 {
    -webkit-text-size-adjust: 100%;
    font-family: Inter,Noto Sans SC,ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji;
    --tw-bg-opacity: 1;
    box-sizing: border-box;
    border: 0 solid #e4e4e7;
    width: 100%;
    text-align: center;
    font-size: .75rem;
    line-height: 1rem;
    font-weight: 500;
    --tw-text-opacity: 1;
    color: rgb(161 161 170/var(--tw-text-opacity));
    }
</style>
<h2>总访问量<span id="busuanzi_value_site_pv"></span>次 总访客数<span id="busuanzi_value_site_uv"></span>人</h2>
</body>
</html>
```

**Reply by waleraye** [reply]
非常感谢,根据您的配置,我设置好了
